crates/typst-svg/src/lib.rs

```diff
@@ -452,6 +452,14 @@ impl SVGRenderer {
             Size::new(Abs::pt(width), Abs::pt(height)),
             self.text_paint_transform(state, &text.fill),
         );
+        if let Some(stroke) = &text.stroke {
+            self.write_stroke(
+                stroke,
+                Size::new(Abs::pt(width), Abs::pt(height)),
+                self.text_paint_transform(state, &text.fill),
```

Comment:
Be careful, here it should be `&stroke.paint` instead of `&text.fill` otherwise the transform is wrong!

Reply:
thanks for figure it out!